mxcube3/ui/components/RemoteAccess/ObserverDialog.jsx

```diff
@@ -48,6 +48,42 @@ export class ObserverDialog extends React.Component {
     return 'Observer mode';
   }
 
+  observerName() {
```

Comment:
is looks as repeated code, can it be merged? You could use the same code and just disable the edtion of the name in case of user login